jsapp/js/mixins.es6

```diff
@@ -242,7 +242,11 @@ mixins.dmix = {
     } else if (uid) {
       actions.resources.loadAsset({id: uid});
     }
-  }
+  },
+  removeSharing: function(evt) {
+    evt.preventDefault();
+    mixins.clickAssets.click.asset.removeSharing(this.props.params.uid);
+  },
```

Comment:
I don't think you need to pass `evt` to this function, let's make it only about removing sharing. `evt` is already `preventDefault`-ed :)